srcpkgs/cryfs/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'cryfs'
+pkgname=cryfs
+version=0.10.0
+revision=1
+build_style=cmake
+configure_args="-DCRYFS_UPDATE_CHECKS=off -DOpenMP_CXX_FLAGS=-I/usr/include
```

Comment:
that will most likely not work in cross compilation

Reply:
it is missing ${XBPS_CROSS_BASE}

Reply:
also there isn't a INCLUDEDIR like there is LIBRARY ?

Reply:
> also there isn't a INCLUDEDIR like there is LIBRARY ?

configure args were taken from [homebrew](https://github.com/Homebrew/homebrew-core/blob/316be93f96463c3182f376a6260cbcc0649134d9/Formula/cryfs.rb#L31), with some guess and check. Passing `-DOpenMP_CXX_FLAGS='-Xpreprocessor -fopenmp` didn't work per upstream, due to whitespacing errors. Build succeeds anyway without those args.

Not sure about `INCLUDEDIR` with cmake. It could be attempted.

Reply:
It is more common to use INCLUDEDIR for passing cross-compilation friendly include directories than to manipulate CXX_FLAGS